librocksdb_sys/crocksdb/c.cc

```diff
@@ -5853,4 +5857,76 @@ void ctitandb_delete_files_in_ranges_cf(
   SaveError(errptr, static_cast<TitanDB*>(db->rep)->DeleteFilesInRanges(
                         cf->rep, &ranges[0], num_ranges, include_end));
 }
+
+/* RocksDB Cloud */
+#ifdef USE_CLOUD
+struct cloud_envoptions_t {
+  CloudEnvOptions rep;
+};
+
+crocksdb_env_t* cloud_env_create() {
```

Comment:
let's name it `crocksdb_cloud_aws_env_create`, and it should has the same list of params as `NewAwsEnv()`.